### ISSUE TITLE
Is there a new title key because the one it used to be doesnt seem to populate the selection list

### ISSUE THREAD

**Issue by LinkX679** [dojafoja/FunKii-UI#12]
Im using the wiiu.titlekeys.gq one

**Comment by dojafoja** [comment]
Yes that domain expired recently and wasn't renewed again. Here is a tip: The site is still alive and well, just not at the same domian. The IP address remains the same so if you search around on Reddit you can find the IP address it looks something like this http://104.129.5x.xxx and it works fine with Funkii-UI. There is also a site that ends with ddns.net that works fine. There is another that ends in .site that isn't supported yet in v2.1.8 however I've been working on a collaborative project with a Spanish team to add multi language and theme support. That one does support the new site ending with .site and can be downloaded from the dark-theme branch here: https://github.com/dojafoja/FunKii-UI/tree/dark_theme Its currently set to Spanish so you would need to switch languages in the Options tab
